### ISSUE TITLE
[plugin-explorer] graphiql-explorer.d.ts raises error `ts(8020)`

### ISSUE THREAD

**Issue by hasparus** [graphql/graphiql#2730]
Howdy 👋 

I've just noticed that _@graphiql/plugin-explorer/types/graphiql-explorer.d.ts_ contains invalid TypeScript and raises `ts(8020)`.

You can notice the error in the following picture.

![image](https://user-images.githubusercontent.com/15332326/187484435-8c85b3f0-d962-43c5-93b8-245d8ea505de.png)

We could easily change this by removing that leading `?` (see below). I'm happy to send a PR if you'd like.

```diff
-    styles?: ?{
+    styles?: {
      explorerActionsStyle?: { [key: string]: any };
      buttonStyle?: { [key: string]: any };
      actionButtonStyle?: { [key: string]: any };
    };
```

**Comment by thomasheyenbrock** [comment]
That's a whoopsie on me 🙈 I copied over the flow type definitions from the `graphiql-explorer` package and converted them to TS. Seems I missed this nullable type annotation 🙈

**Comment by thomasheyenbrock** [comment]
Just released the fix for this in `@graphiql/plugin-explorer@0.1.1`
